### PR TITLE
[MM-60215] Webapp resize with announcement banner enabled will cause screen to blank out

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Summary
- Doesn't effect changes which were made in https://github.com/mattermost/mattermost/pull/27171

CleanShot 2024-08-27 at 19 02 26@2x

CleanShot 2024-08-27 at 19 01 39@2x

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-60215

#### Screenshots

|  before  |  after  |
|----|----|
| CleanShot 2024-08-27 at 19 07 45@2x | CleanShot 2024-08-27 at 19 03 04@2x |

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/sass/base/_structure.scss [hunk 1]
```diff
@@ -234,8 +241,13 @@ body.app__body #root {
             padding-right: 0;
             z-index: 17;
             grid-template:
-                'announcement'min-content 'header'min-content 'main'auto 'footer'min-content / auto;
-
+                'announcement' min-content
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/sass/base/_structure.scss:244]: This is the main change. Other changes are such that its easier to see the columns and rows in grid layout.

File: webapp/channels/src/sass/base/_structure.scss [hunk 2]
```diff
@@ -234,8 +241,13 @@ body.app__body #root {
...
+                'admin-announcement' min-content
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/sass/base/_structure.scss:245]: This is the main change. Other changes are such that its easier to see the columns and rows in grid layout.

**Review by larkox — APPROVED** [review]

**Review by asaadmahmood — APPROVED** [review]
LGTM but I think there probably is a reason why those classes were added in the first place. Just adding @matthewbirtch as a reviewer to see if he remembers.

**Review by matthewbirtch — APPROVED** [review]
Thanks @M-ZubairAhmed. Fix looks good to me

**Comment by M-ZubairAhmed** [comment]
@asaadmahmood i didnt a new class

**Comment by asaadmahmood** [comment]
@M-ZubairAhmed I meant the grid values.

**Comment by amyblais** [comment]
@M-ZubairAhmed Can you help with a manual cherry-pick for ``release-9.10`` branch?

**Comment by amyblais** [comment]
@M-ZubairAhmed Would you be able to help cherry-pick this to ``release-9.11`` branch?

**Pull request merged**